roles/rsyslog/defaults/main.yaml

```diff
@@ -291,60 +281,29 @@ rsyslog_weight_map:
   'input': '90'
   'inputs': '90'
 
-# rsyslog_rules
-#
-# List of YAML dictionaries, each dictionary should contain ``rsyslogd``
-# configuration in a special format. See :ref:`rsyslog_rules` for more
-# details. This list should be used for configuration of all hosts in the
-# inventory.
-rsyslog_rules: []
-
-# rsyslog_group_rules
-#
-# List of YAML dictionaries, each dictionary should contain ``rsyslogd``
-# configuration in a special format. See :ref:`rsyslog_rules` for more
-# details. This list should be used for configuration of a group of hosts in
-# the inventory.
-rsyslog_group_rules: []
-
-# rsyslog_host_rules
-#
-# List of YAML dictionaries, each dictionary should contain ``rsyslogd``
-# configuration in a special format. See :ref:`rsyslog_rules` for more
-# details. This list should be used for configuration of specific hosts in the
-# inventory.
-rsyslog_host_rules: []
-
-# rsyslog_dependent_rules
+# __rsyslog_.*_rules
 #
 # List of YAML dictionaries, each dictionary should contain ``rsyslogd``
-# configuration in a special format. See :ref:`rsyslog_rules` for more
-# details. This list should be used for configuration by other Ansible roles.
-rsyslog_dependent_rules: []
-
-# rsyslog_.*_rules
-#
-# List of YAML dictionaries, each dictionary should contain ``rsyslogd``
-# configuration in a special format. See :ref:`rsyslog_rules` for more
-# details. This lis specifies default ``rsyslogd`` configuration enabled in the
-# role.
-rsyslog_common_rules:
+# configuration in a special format. This lis specifies default ``rsyslogd``
+# configuration enabled in the role.
+__rsyslog_common_rules:
```

Comment:
If you are prepending underscores to variable names, does it mean that the variables are internal and therefore don't need to be settable by users? - but then they belong to vars and not defaults, no?

Also I would say that this diverges from the topic of fixing package module related issues.

Reply:
> If you are prepending underscores to variable names, does it mean that the variables are internal and therefore don't need to be settable by users? - but then they belong to vars and not defaults, no?

That's correct.  They should belong to vars.

> Also I would say that this diverges from the topic of fixing package module related issues.

+1.  May I stop this here and move to another PR for the internal var fixes (including docs)?

Reply:
yes please, finish the package related changes in this PR and continue variable renaming (if not related to the package module changes) in another PR.

Reply:
and please rename `rsyslog_packages` to `rsyslog_extra_packages` instead of to `rsyslog_additional_packages`, the former seems more idiomatic to me.

Reply:
Done.